lib/FeatureSetUtils/Utils/FeatureSetBuilder.py

```diff
@@ -337,7 +337,7 @@ def _filter_expression_matrix(self, expression_matrix_ref, feature_ids,
         filtered_expression_matrix_data['diff_expr_matrix_ref'] = diff_expression_matrix_ref
         filtered_expression_matrix_data['data'] = filtered_data
 
-        object_type = 'KBaseFeatureValues.ExpressionMatrix'
+        object_type = expression_matrix_info[2]
```

Comment:
This is fine. But do we want to get ride of type version and save as the latest object?